tests/baselines/reference/api/typescript.d.ts

```diff
@@ -3543,6 +3550,8 @@ declare namespace ts {
     function getJSDocProtectedTag(node: Node): JSDocProtectedTag | undefined;
     /** Gets the JSDoc protected tag for the node if present */
     function getJSDocReadonlyTag(node: Node): JSDocReadonlyTag | undefined;
+    /** Gets the JSDoc deprecated tag for the node if present */
+    function getJSDocDeprecatedTagNoCache(node: Node): JSDocDeprecatedTag | undefined;
```

Comment:
Is this actually non-caching? The code is the same as I remember the others right next to it, which are not labeled non-caching. (I’ll have to check on my computer because the git hub app doesn’t let me expand nearby code. )